src/containers/dnsmasqConfig.js

```diff
@@ -1,25 +1,28 @@
-export default {
+export default (port = '53') => ({
   Image: 'andyshinn/dnsmasq:latest',
   name: 'dotdocker-dnsmasq',
+  Labels: {
+    port,
+  },
   Cmd: ['--address=/docker/127.0.0.1', '--log-facility=-'],
   ExposedPorts: {
-    '53/tcp': {},
-    '53/udp': {},
+    [`${port}/tcp`]: {},
+    [`${port}/udp`]: {},
   },
   HostConfig: {
     PortBindings: {
       '53/tcp': [
         {
-          HostPort: '53',
+          HostPort: `${port}`,
         },
       ],
       '53/udp': [
         {
-          HostPort: '53',
+          HostPort: `${port}`,
```

Comment:
do we need to wrap this in a template string at all?

Reply:
no